apps/mull-ui/src/app/components/profile-header/profile-header.tsx

```diff
@@ -0,0 +1,88 @@
+import { faShareAlt } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { MullButton } from '..';
+import './profile-header.scss';
+
+function checkFriendStatus(isFriend: boolean) {
+  let friendStatus: string;
+  if (isFriend === true) {
+    friendStatus = 'Friends';
+  }
+  if (isFriend === false) {
+    friendStatus = 'Pending';
+  }
+  // No relation in the Friends table
+  if (isFriend === null) {
+    friendStatus = 'Add Friend';
+  }
+  return (
+    <div className="friend-status-container">
+      <MullButton className={isFriend ? 'friend-status-button' : 'not-friend-status-button'}>
+        {friendStatus}
+      </MullButton>
+    </div>
+  );
+}
```

Comment:
I think you should bring this function into the component. Also, a switch statement could be good to replace all the if's

Reply:
In the first implementation, this function was a [switch statement](https://github.com/RGPosadas/Mull/blob/9958ad656a80d0639a99c194aed0c07773f2feed/apps/mull-ui/src/app/components/profile-header/profile-header.tsx#L11-L29), but after refactoring with @r-saba, it became an if statement since it looked cleaner. Take a look and tell me what you think about either statement.

Reply:
In that case, I think it's fine, just make the middle condition an else-if and the last one an else for better performance